lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -384,15 +426,29 @@ private function getPropertyImports(ReflectionProperty $property)
         return array_merge($classImports, $traitImports);
     }
 
+    private function getFunctionImports(ReflectionFunction $function) : array
```

Comment:
This function shares the same logic as `getClassImports()` except an array key. Can you please outsource the common logic into another private method?

Reply:
I've merge `getClassImports` and `getFunctionImports` into an unique `getImports` method.